src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -351,6 +351,8 @@ public struct StartConfiguration
             public bool StartInTestMode { get; set; }
             public IUpdateManager UpdateManager { get; set; }
             public ISchedulerThread SchedulerThread { get; set; }
+            public IAuthProvider AuthProvider { get; set; }
+            public string PackageManagerAddress { get; set; }
```

Comment:
OK - for something like this, a class with readonly backing store fiends is generally preferable. readonly in C# lets them be set once, so it doesn't stop the use of initialisers. I don't know off the top of my head whether they play nicely with auto-props.

However it's not a big deal in this case.